awesome_classifier.py

```diff
@@ -0,0 +1,35 @@
+from torchvision import models
+import torch
+dir(models)
+
+alexnet = models.alexnet(pretrained=True)
+
+from torchvision import transforms
+
+#changing the dimensions of the image.
```

Comment:
This can be declared inside in a function. Remember Modularity is a great factor in making software more readable and manageable